src/groups/bmq/bmqp/bmqp_confirmeventbuilder.h

```diff
@@ -170,7 +185,7 @@ inline int ConfirmEventBuilder::eventSize() const
         return 0;  // RETURN
     }
 
```

Comment:
We might avoid this check since there are no situations when we allow this pointer to be 0. We always set a new valid pointer to a blob in `reset()`. We always build with SAFE assert level enabled and we might want to avoid some runtime checks like this.